test/integration/contributionsController.test.js

```diff
@@ -0,0 +1,155 @@
+const chai = require('chai')
+const sinon = require('sinon')
+const { expect } = chai
+const chaiHttp = require('chai-http')
+
+const authService = require('../../services/authService')
+const githubService = require('../../services/githubService')
+const addUser = require('../utils/addUser')
+
+const app = require('../../server')
+
+chai.use(chaiHttp)
+
+// Import fixtures
+const githubPRInfo = require('../fixtures/contributions/githubPRInfo')()
+
+let jwt
+
+describe('Contributions route', function () {
+  describe('Create User', function () {
+    before(async function () {
+      const userId = await addUser()
+      jwt = authService.generateAuthToken({ userId })
+    })
+
+    afterEach(function () {
+      sinon.restore()
```

Comment:
This is not required as you are not mocking any function calls.

Reply:
Github api calls are mocked